packages/esm-patient-immunizations-app/src/immunizations/immunizations-form.component.tsx

```diff
@@ -119,14 +132,40 @@ const ImmunizationsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, close
     updateSingle('currentDose', currentDose);
   };
 
+  const vaccines = mockPaginatedImmunizations;
+  const VaccineSelect = () => (
+    <Select
+      id="vaccine"
+      invalidText="Required"
+      labelText=""
+      onChange={(event) => updateSingle('vaccineName', event.target.value)}
+      value={formState.vaccineName}
+    >
+      {!formState.vaccineName ? <SelectItem text={t('chooseVaccine', 'Choose a vaccine')} value="" /> : null}
```

Comment:
IIRC, there's a disabled property that should be set for a default but invalid drop-down option.